apps/server/src/events/event.e2e.spec.ts

```diff
@@ -63,10 +63,10 @@ describe('[E2E] Request', () => {
 
     expect(await test.fetch(`/events/${eventId}`, { token })).toHaveProperty('body.body', '<p>body</p>');
 
-    expect(await test.application.getMemberNotifications({ memberId: test.member.id })).toHaveProperty(
-      'notifications.0',
-      expect.objectContaining<Partial<shared.Notification>>({ title: 'Nouvel événement', content: 'title' }),
-    );
+    // expect(await test.application.getMemberNotifications({ memberId: test.member.id })).toHaveProperty(
```

Comment:
**question**

any reason why not deleting this commenting code?

Reply:
Yes, we should uncomment it. I commented it when the query was using the old table, and probably forgot to uncomment it.

Reply:
fc82172